packages/usa-character-count/src/index.js

```diff
@@ -178,11 +213,19 @@ const characterCount = behavior(
         updateCountMessage(this);
       },
     },
+    focusout: {
+      [INPUT]() {
+        toggleErrorState(this);
+      },
+    },
```

Comment:
## Add form group error classes on focusout

This idea came from a discussion between @jaclinec and I. We wanted to add the full error state classes but didn't want to have the content shift while the user was typing. She suggested we add these classes only after the user _leaves the field_.

Reply:
@amyleadem this is the intentional behavior based on @jaclinec's recommendation to not cause content shift while the user is typing (see [comment](https://github.com/uswds/uswds/pull/5908#discussion_r1587818882) above and testing instructions)

If the content shift on _removal_ is acceptable, I can make it so that the error state is immediately removed rather than waiting for `focusout`.

Here's a quick example of adding/removing the classes on `input`. I think this would be particularly jarring for screen magnification users

 
[Kapture 2024-05-23 at 10.02.35.webm](https://github.com/uswds/uswds/assets/25211650/b35365c9-3e71-47e4-8692-aa7191e8292f)

Reply:
Behavior updated in [b7fa1f0](https://github.com/uswds/uswds/pull/5908/commits/b7fa1f0494c815f2c04032cf025d16944ea6912c)

Preview of new behaviors from the [demo repo](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/cm-test-repo-5908/iframe.html?args=&id=components-form-inputs-character-count--character-count&viewMode=story). 

[Kapture 2024-08-23 at 12.16.41.webm](https://github.com/user-attachments/assets/2061a434-b21d-4e3f-85b6-b88cd4d34472)


There is a slight shift due to the increased border, but I don't think it's enough to cause concern!

Reply:
Behavior removed to account for new error state styles

Reply:
This behavior makes more sense to me. Thanks @mahoneycm!